R/zzz.R

```diff
@@ -55,20 +55,28 @@ check_time <- function(time) {
 }
 
 
-get_widget <- function(comparison_item, category, gprop) {
+get_widget <- function(comparison_item, category, gprop, 
+                       tz = "300", 
+                       hl = "en-US") {
```

Comment:
need validation for `hl` to ensure the provided value exists.

Reply:
The gtrends function already has this validation, and the get_widget () should not be called outside the scope of gtrends so I believe that further validation is not necessary. It is in line 108.